src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -82,6 +82,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       request.getRequestDispatcher("/WEB-INF/view/login.jsp").forward(request, response);
       return;
     }
+    
+    /** Allows to know if the user is admin in other pages */
+    if(user.isAdmin())
```

Comment:
this is a bit of a [nit](https://www.techopedia.com/definition/31651/nit) but having the opening and closing brackets in these one line if-statements is a bit better for readability.

Reply:
Also, I have a preference for adding the comment/javadoc to the method itself instead of the line of code. Code is usually pretty self-documenting, but if people really want to know what a method does they will look at the method javadoc. Adding it here makes it slightly less reusable (say if you call "isAdmin" from anywhere else).